proto/rill/admin/v1/api.proto

```diff
@@ -184,6 +184,11 @@ service AdminService {
     };
   }
 
+  // RequestRepresentativeAuthToken returns the temporary token for given email
+  rpc RequestRepresentativeAuthToken(RequestRepresentativeAuthTokenRequest) returns (RequestRepresentativeAuthTokenResponse) {
+    option (google.api.http) = {get: "/v1/tokens/{email}"};
+  }
```

Comment:
1. Refactor to `IssueRepresentativeAuthToken`
2. Change `get` to `post` (then also need to pass `body: "*"`)
3. Change path to `/v1/tokens/represent`